src/lib/contributers.js

```diff
@@ -1,50 +1,66 @@
-import React, { useState, useEffect } from 'react'
-import Axios from 'axios'
-import adminList from '../data/adminList'
-import mentorList from '../data/mentorList'
+import React, { useState, useEffect } from 'react';
+import Axios from 'axios';
+import adminList from '../data/adminList';
+import mentorList from '../data/mentorList';
+import Card from '../components/Card';
 
+const useContributerData = (props) => {
+  const URL =
+    'https://api.github.com/repos/anushbhatia/Social-Scheduler/stats/contributors';
+  const [data, setData] = useState([]);
+  const [err, seterr] = useState(null);
 
-const useContributerData = props => {
-    const URL = 'https://api.github.com/repos/anushbhatia/Social-Scheduler/stats/contributors'
-    const [data, setData] = useState([]);
-    const [err, seterr] = useState(null);
+  const response = async () => {
+    try {
+      let res = await Axios({
+        method: 'GET',
+        url: URL,
+      });
 
-    const response = async () => {
-        try {
-            let res = await Axios({
-                method: "GET",
-                url: URL
-            });
+      let contribtorList = [];
 
-            let contribtorList = []
+      res.data.map((el) => {
+        const { id, login, avatar_url, html_url } = el.author;
 
-            res.data.map(el => {
-
-                const { id, login, avatar_url, html_url } = el.author
-
-                if(!adminList.includes(login) && !mentorList.includes(login)){
-                    contribtorList.push({
-                        id: id,
-                        username: login,
-                        avatar: avatar_url,
-                        profile: html_url
-                    })
-                }
-
-            })
-
-            setData(contribtorList)
-        } catch (error) {
-            seterr(error);
+        if (!adminList.includes(login) && !mentorList.includes(login)) {
+          contribtorList.push({
+            id: id,
+            username: login,
+            avatar: avatar_url,
+            profile: html_url,
+          });
         }
+      });
+      const data = [];
+      for (let i = 0; i < contribtorList.length; i = i + 3) {
+        data.push(contribtorList.slice(i, i + 3));
+      }
```

Comment:
What's the use for this??

Reply:
Styling was done previously, so I took this approach. I will make necessary changes to css file and commit again.